tonic-build/src/lib.rs

```diff
@@ -155,6 +155,50 @@ pub trait Method {
     ) -> (TokenStream, TokenStream);
 }
 
+/// Attributes that will be added to `mod` and `struct` items. Should be pairs of (`pattern`, `attribute`).
+#[derive(Debug, Default, Clone)]
+pub struct Attributes {
+    /// `mod` attributes.
+    pub module: Vec<(String, String)>,
+    /// `struct` attributes.
+    pub structure: Vec<(String, String)>,
+}
```

Comment:
Should we perhaps add `#[non_exhaustive]` so we can add new fields without breakage?

Reply:
good idea! adding now